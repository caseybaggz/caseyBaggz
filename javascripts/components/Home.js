import React from 'react';
import classNames from 'classnames';
import NavLink from './NavLink';
import Footer from './Footer';


export default class Home extends React.Component {
  render() {
    return (
      <div className="home-view-container">
        <div className="feature">
          <p className="sub-headline">
            React & Rails
            <span className="br-mobile" />
            Creative Developer
          </p>
        </div>

        <div className="recent-work-container">
          <p className="headline">recent work</p>
          <div className="work-wrapper">
            <div className="recent-item lays">
              <NavLink to="/works/lays" className="link">
                <span className="info">
                  <p className="type">React, es6 & Rails</p>
                </span>
              </NavLink>
            </div>

            <div className="recent-item thrive">
              <NavLink to="/works/thrive" className="link">
                <span className="info">
                  <p className="type">React, es6 & Rails</p>
                </span>
              </NavLink>
            </div>
          </div>
        </div>

        <div className="cta-wrapper">
          <div className="btn-wrapper">
            <a href="mailto:caseybaggz@gmail.com" className="btn">let's talk</a>
          </div>
        </div>

        <Footer />
      </div>
    );
  }


  // PRIVATE

}
