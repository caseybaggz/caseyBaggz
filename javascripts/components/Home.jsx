import React from 'react';
import classNames from 'classnames';
import NavLink from '../components/NavLink';


export default class Home extends React.Component {
  render() {
    return (
      <div className="home-view-container">
        <div className="feature">
          <p className="sub-headline">
            React & Rails
            <span className="br-mobile" />
            Creative Developer
          </p>
        </div>

        <div className="recent-work-container">
          <p className="headline">recent work</p>
          <div className="work-wrapper">
            <div className="recent-item lays">
              <NavLink to="/works/lays" className="link">
                <span className="info">
                  <p className="type">React, es6 & Rails</p>
                </span>
              </NavLink>
            </div>

            <div className="recent-item thrive">
              <NavLink to="/works/thrive" className="link">
                <span className="info">
                  <p className="type">React, es6 & Rails</p>
                </span>
              </NavLink>
            </div>
          </div>
        </div>

        <div className="cta-wrapper">
          <div className="btn-wrapper">
            <a href="mailto:caseybaggz@gmail.com" className="btn">let's talk</a>
          </div>
        </div>

        <footer className="footer">
          <ul className="social-list">
            <li className="item github">
              <a href="https://github.com/caseybaggz" className="link" target="_blank" />
            </li>
            <li className="item linkedin">
              <a href="https://www.linkedin.com/in/caseybaggz?trk=hp-identity-name" className="link" target="_blank" />
            </li>
            <li className="item twitter">
              <a href="https://twitter.com/caseybaggz" className="link" target="_blank" />
            </li>
            <li className="item codepen">
              <a href="http://codepen.io/caseybaggz/" className="link" target="_blank" />
            </li>
          </ul>
        </footer>
      </div>
    );
  }


  // PRIVATE

}
