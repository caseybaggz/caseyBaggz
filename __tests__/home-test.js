import React from 'react';
import Home from '../javascripts/components/Home';
import renderer from 'react-test-renderer';


it('should render Home component successfully', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
