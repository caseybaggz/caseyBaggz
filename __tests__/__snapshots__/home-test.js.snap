exports[`test should render Home component successfully 1`] = `
<div
  className="home-view-container">
  <div
    className="feature">
    <p
      className="sub-headline">
      React & Rails
      <span
        className="br-mobile" />
      Creative Developer
    </p>
  </div>
  <div
    className="recent-work-container">
    <p
      className="headline">
      recent work
    </p>
    <div
      className="work-wrapper">
      <div
        className="recent-item lays">
        <a
          className="link"
          onClick={[Function bound handleClick]}
          style={Object {}}>
          <span
            className="info">
            <p
              className="type">
              React, es6 & Rails
            </p>
          </span>
        </a>
      </div>
      <div
        className="recent-item thrive">
        <a
          className="link"
          onClick={[Function bound handleClick]}
          style={Object {}}>
          <span
            className="info">
            <p
              className="type">
              React, es6 & Rails
            </p>
          </span>
        </a>
      </div>
    </div>
  </div>
  <div
    className="cta-wrapper">
    <div
      className="btn-wrapper">
      <a
        className="btn"
        href="mailto:caseybaggz@gmail.com">
        let\'s talk
      </a>
    </div>
  </div>
  <footer
    className="footer-container">
    <ul
      className="social-list">
      <li
        className="item github">
        <a
          className="link"
          href="https://github.com/caseybaggz"
          target="_blank" />
      </li>
      <li
        className="item linkedin">
        <a
          className="link"
          href="https://www.linkedin.com/in/caseybaggz"
          target="_blank" />
      </li>
      <li
        className="item twitter">
        <a
          className="link"
          href="https://twitter.com/caseybaggz"
          target="_blank" />
      </li>
      <li
        className="item codepen">
        <a
          className="link"
          href="http://codepen.io/caseybaggz/"
          target="_blank" />
      </li>
    </ul>
  </footer>
</div>
`;
